src/components/tx/TxSimulation/useSimulation.ts

```diff
@@ -38,17 +38,25 @@ export const useSimulation = (): UseSimulationReturn => {
   }, [])
 
   const simulateTransaction = useCallback(
-    async (params: Omit<SimulationTxParams, 'gasLimit'>) => {
+    async (params: Omit<SimulationTxParams, 'gasLimit'> & { manualGasLimit?: number }) => {
```

Comment:
You can set `SingleTransactionSimulationParams['gasLimit']` and `MultiSendTransactionSimulationParams['gasLimit']` to optional and then use `params` directly.

Reply:
Done!